### PR TITLE
Update references for known issues back to CHANGELOG.md (release notes).

### PR DESCRIPTION
**Pull request by david-mcmahon**
Also be sure to delete http://kubernetes.io/docs/user-guide/known-issues/

### CODE REVIEW

**Comment by david-mcmahon** [comment]
@bgrant0607 I assume you need to do something here with the do-not-merge label?

File: docs/user-guide/known-issues.md [hunk 1]
```diff
@@ -1,40 +0,0 @@
...
-Documentation for other releases can be found at
-[releases.k8s.io](http://releases.k8s.io).
-</strong>
---
-
-<!-- END STRIP_FOR_RELEASE -->
-
-<!-- END MUNGE: UNVERSIONED_WARNING -->
-
-This file has moved to: http://kubernetes.github.io/docs/user-guide/known-issues/
```
Comment (bgrant0607) [inline docs/user-guide/known-issues.md]: Please link to release notes instead of deleting this file.

File: docs/README.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ Documentation for other releases can be found at
   folder.
 
 * If something went wrong, see the [troubleshooting](troubleshooting.md) document for how to debug.
-You should also check the [known issues](user-guide/known-issues.md) for the release you're using.
+You should also check the known issues for the [release](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md) you're using.
```
Comment (bgrant0607) [inline docs/README.md]: Please use a relative link: ../CHANGELOG.md

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ Sometimes things go wrong.  This guide is aimed at making them right.  It has tw
    * [Troubleshooting your application](user-guide/application-troubleshooting.md) - Useful for users who are deploying code into Kubernetes and wondering why it is not working.
    * [Troubleshooting your cluster](admin/cluster-troubleshooting.md) - Useful for cluster administrators and people whose Kubernetes cluster is unhappy.
 
-You should also check the [known issues](user-guide/known-issues.md) for the release you're using.
+You should also check the known issues for the [release](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG.md) you're using.
```
Comment (bgrant0607) [inline docs/troubleshooting.md]: Please use a relative link: ../CHANGELOG.md

**Comment by david-mcmahon** [comment]
PTAL

**Comment by bgrant0607** [comment]
LGTM. Manually merging due to touching a blocking file. cc @k8s-oncall

**Pull request merged**